slsDetectorServers/eigerDetectorServer/slsDetectorFunctionList.c

```diff
@@ -322,15 +302,22 @@ u_int32_t getDetectorIP() {
 void initControlServer() {
     LOG(logINFOBLUE, ("Configuring Control server\n"));
     if (!updateFlag && initError == OK) {
-        readDetectorNumber();
+#ifndef VIRTUAL
```

Comment:
Why is there a difference between virtual and normal for getting the modid?

Reply:
We can set module id only in Beb, which is not there in the virtual server. So, I just getModuleId as well but do not assign it to anything, as there is nothing to set for virtual servers.

Reply:
Or should i set it to a variable which is picked up when streaming out udp packets?